buddyboss/buddyboss-platform
# Forums page and Group Discussions are not working with non-english

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #1159 .

### How to test the changes in this Pull Request:

1. Create a page with non-English characters
2. Set it as the main forum page
3. Go to BuddyBoss > settings > Forums > and change the slugs in Single Forum Slugs and Forum Profile Slugs with Non-English characters as well.
4. Go to any forums, topics, or group discussions and notice that it will redirect to 404 page.

### Proof Screenshots or Video
http://prntscr.com/tj0wxx
http://prntscr.com/tj0x5u
http://prntscr.com/tj0xdb
http://prntscr.com/tj0xls




### Other information:
